### DISCUSSION TITLE
Roc Mining Überlappende Texte

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#560] [Übersetzungsvorschläge]
Beim Roc mining habe ich überlappende Texte gefunden. Ich glaube ein Teil der Texte überlappten sich auch in der Prospector, die wir schon behoben haben.

![ScreenShot-2024-07-03_12-42-57-806](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/f12e3c4e-04e6-4f66-b369-c4687810a753)

**Comment by rjcncpt** [comment]
> Beim Roc mining habe ich überlappende Texte gefunden. Ich glaube ein Teil der Texte überlappten sich auch in der Prospector, die wir schon behoben haben.

Moin. Das muss neu sein. Auch das plötzlich alles in CAPS steht, irritiert mich. Ich glaube e tatsächlich das CIG entweder das UI gerade umbaut oder komplett zerhackt
